diff --git a/node_modules/@nebular/theme/bundles/theme.umd.js b/node_modules/@nebular/theme/bundles/theme.umd.js
index 954eb08..43e4e02 100644
--- a/node_modules/@nebular/theme/bundles/theme.umd.js
+++ b/node_modules/@nebular/theme/bundles/theme.umd.js
@@ -1057,7 +1057,7 @@ var NbOverlayContainerAdapter = /** @class */ (function (_super) {
 var NbViewportRulerAdapter = /** @class */ (function (_super) {
     tslib.__extends(NbViewportRulerAdapter, _super);
     function NbViewportRulerAdapter(platform, ngZone, ruler, scroll) {
-        var _this = _super.call(this, platform, ngZone) || this;
+        var _this = _super.call(this, platform, ngZone, document) || this;
         _this.ruler = ruler;
         _this.scroll = scroll;
         return _this;
diff --git a/node_modules/@nebular/theme/esm2015/components/cdk/adapter/viewport-ruler-adapter.js b/node_modules/@nebular/theme/esm2015/components/cdk/adapter/viewport-ruler-adapter.js
index 2485b14..58884ad 100644
--- a/node_modules/@nebular/theme/esm2015/components/cdk/adapter/viewport-ruler-adapter.js
+++ b/node_modules/@nebular/theme/esm2015/components/cdk/adapter/viewport-ruler-adapter.js
@@ -6,7 +6,7 @@ import { NbLayoutRulerService } from '../../../services/ruler.service';
 import { NbLayoutScrollService } from '../../../services/scroll.service';
 export class NbViewportRulerAdapter extends ViewportRuler {
     constructor(platform, ngZone, ruler, scroll) {
-        super(platform, ngZone);
+        super(platform, ngZone, document);
         this.ruler = ruler;
         this.scroll = scroll;
     }
diff --git a/node_modules/@nebular/theme/fesm2015/index.js b/node_modules/@nebular/theme/fesm2015/index.js
index ac1be43..a04dbf0 100644
--- a/node_modules/@nebular/theme/fesm2015/index.js
+++ b/node_modules/@nebular/theme/fesm2015/index.js
@@ -959,7 +959,7 @@ NbOverlayContainerAdapter.decorators = [
 
 class NbViewportRulerAdapter extends ViewportRuler {
     constructor(platform, ngZone, ruler, scroll) {
-        super(platform, ngZone);
+        super(platform, ngZone, document);
         this.ruler = ruler;
         this.scroll = scroll;
     }
diff --git a/node_modules/@nebular/theme/schematics/util/ast.d.ts b/node_modules/@nebular/theme/schematics/util/ast.d.ts
index 68226f6..c936383 100644
--- a/node_modules/@nebular/theme/schematics/util/ast.d.ts
+++ b/node_modules/@nebular/theme/schematics/util/ast.d.ts
@@ -1,4 +1,4 @@
 import { Tree } from '@angular-devkit/schematics';
-import { WorkspaceProject } from '@angular-devkit/core/src/experimental/workspace';
-export declare function isImportedInMainModule(tree: Tree, project: WorkspaceProject, moduleName: string): boolean;
+import { ProjectDefinition  } from '@angular-devkit/core/src/workspace';
+export declare function isImportedInMainModule(tree: Tree, project: ProjectDefinition, moduleName: string): boolean;
 export declare function getAppModulePath(host: Tree, mainPath: string): string;
diff --git a/node_modules/@nebular/theme/schematics/util/project.d.ts b/node_modules/@nebular/theme/schematics/util/project.d.ts
index e47b39c..f25bb17 100644
--- a/node_modules/@nebular/theme/schematics/util/project.d.ts
+++ b/node_modules/@nebular/theme/schematics/util/project.d.ts
@@ -1,6 +1,6 @@
 import { Tree } from '@angular-devkit/schematics';
-import { WorkspaceProject } from '@angular-devkit/core/src/experimental/workspace';
+import { ProjectDefinition  } from '@angular-devkit/core/src/workspace';
 /**
  * Gets project workspace from the specified tree by given project name
  * */
-export declare function getProject(tree: Tree, projectName: string): WorkspaceProject;
+export declare function getProject(tree: Tree, projectName: string): ProjectDefinition;
